VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -252,4 +252,12 @@ export class VideoCallService {
         });
         await this.apiClient.callWitness(conferenceId, participantId).toPromise();
     }
+
+    async dismissParticipantFromHearing(conferenceId: string, participantId: string) {
+        this.logger.info(`${this.loggerPrefix} Attempting to dismiss participant from hearing`, {
+            conference: conferenceId,
+            participant: participantId
+        });
+        await this.apiClient.dismissWitness(conferenceId, participantId).toPromise();
```

Comment:
why toPromise? not returning?

Reply:
The global error handler will capture errors and log it but actually maybe we should capture it to when the request fails and we want to show a message of some nature